lib/lich/gameobj.rb

```diff
@@ -0,0 +1,341 @@
+require 'rexml/document'
+
+# GameObj#load_data depends on this monkey patch
```

Comment:
As much as I dislike polluting NilClass globally with this monkey patch, it seemed better to include it than to modify the GameObj implementation below.

I've never used a [refinement](https://blog.codeship.com/ruby-refinements/) in real code, but that could be an alternative way to deal with this monkey patch without polluting NilClass everywhere.